src/main.rs

```diff
@@ -109,8 +114,15 @@ fn main() -> Result<()> {
                 .short("a")
                 .long("ascii_language")
                 .takes_value(true)
-                .default_value("")
-                .help("Overrides showing the dominant language ascii logo"),
+                .default_value("rust")
```

Comment:
Why is the default value `"rust"`? Just ran your branch for a CoffeeScript project, and it incorrectly used Rust's ASCII art.

It might be better if there is no default value at all, actually. The logic can be:
```rust
let ascii_language = if let Some(ascii) = matches.value_of("ascii_language") {
    // Use user-selected ascii art
} else {
  // Calculate ascii art
}
```
Since `matches.value_of("ascii_language").unwrap()` would often panic with no default value.